traffic_ops/app/db/migrations/2022041810412200_cdni_capacity_limit_with_scopes.down.sql

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/* Downgrading this will lose some data because the specification has changed.
+ * Previously, only total and host limits were set. Now with the scope field, it is not as limited so downgrading
+ * cannot put the data back in the old tables since it wont necessarily fit into those buckets.
+*/
+CREATE TABLE IF NOT EXISTS cdni_total_limits (
```

Comment:
We need to create things in the `public` schema explicitly. Hank Beatty has had issues with our migrations and queries not doing that in the past, I think.